klog_test.go

```diff
@@ -1921,3 +1921,65 @@ func TestStopFlushDaemon(t *testing.T) {
 		t.Error("expected flushD to be stopped")
 	}
 }
+
+func TestBackTrace(t *testing.T) {
+	const skippedNumLines = 2
+	const sizeNumLines = 1
+	const numLinesPerTrace = 2
+
+	_, callerFile, _, _ := runtime.Caller(0)
+	s1 := fmt.Sprintf("%v", Backtrace())
+	normalLines := strings.Count(s1, "\n")
+	if strings.Contains(s1, callerFile) == false {
+		t.Errorf("Expected to see the file, having callsite, in trace %v\n", callerFile)
+	}
+
+	// skip the caller file i.e., the current file where backtrace API is called
+	s2 := fmt.Sprintf("%v", Backtrace(BacktraceSkip(skippedNumLines)))
```

Comment:
Why is `skippedNumLines` 2 and not 1? Why does it skip "lines" and not stack entries?

I wouldn't use a name constant here and just write "1".

Reply:
Update: it looks like 2 was chosen to match the implementation, but the implementation is wrong (starts with its own code).